src/frontends/tensorflow/src/helper_transforms/embedding_segments_feature_fusing.cpp

```diff
@@ -103,10 +103,15 @@ ov::frontend::tensorflow::pass::EmbeddingSegmentSingleFeatureFusion::EmbeddingSe
     auto const_one = make_shared<Constant>(element::i32, Shape{1}, 1);
     auto new_subshape = make_shared<Broadcast>(const_one, num_new_axes);
     auto cond_shape = make_shared<ShapeOf>(tile, element::i32);
-    auto new_cond_shape = make_shared<Concat>(OutputVector{cond_shape, new_subshape}, 0);
+    // use extra dimensions in the begin to avoid concatenation of empty tensors that is not supported by Concat
```

Comment:
can we just check that tensors are empty here?
anyway, it looks like a workaround, we need to solve this issue with empty tensors more generally.
does our specification describe the behavior of Concat op in this case?

Reply:
Some transformation in the common part removes input with empty tensors to `Concat` so it is remained with no inputs.
The problem is in the transformation.

Reply:
Agreed to create a ticket for the common part and fix it there. Once the fix is done, this workaround will be removed.

Reply:
Done